src/test/java/org/aist/aide/generaltypeservice/unit/PatternCrudServiceTests.java

```diff
@@ -0,0 +1,199 @@
+package org.aist.aide.generaltypeservice.unit;
+
+import org.aist.aide.generaltypeservice.domain.exceptions.NotFoundException;
```

Comment:
If checkstyle is already active these imports will start having some fun with the order.